Makefile

```diff
@@ -86,73 +83,31 @@ LAUNCH4J:=~/java/launch4j/launch4j
 vpath %.class $(shell find $(CLASSDIR) -type d)
 vpath %.java  $(shell find $(SRCDIR) -type d -name .svn -prune -o -print)
```

Comment:
You could remove the above five lines (`#SOURCES` to `vpath`).

Reply:
I have no idea what vpath does, was afraid to break anything.

Reply:
It has to do with where `make` searches for files. It's not something one normally needs, outside of unusual use cases like this one.